projects/arlas-components/src/lib/components/mapgl/model/AbstractArlasMapGL.ts

```diff
@@ -466,5 +595,24 @@ export abstract class AbstractArlasMapGL {
     this._eventSubscription.forEach(s => s.unsubscribe());
   }
 
+  public getBounds(){
+    return this.getMapProvider().getBounds();
```

Comment:
I am not sure all implementations of the map will have this getBounds method, but it can evolve in the future.

Reply:
I agree, should be implemented this way in mapboxgl only

Reply:
fixed